interp/src/debugger/parser/command_parser.rs

```diff
@@ -54,8 +54,12 @@ impl CommandParser {
         Ok(Command::Exit)
     }
 
-    fn comm_where(_input: Node) -> ParseResult<Command> {
-        Ok(Command::PrintPC)
+    fn comm_where(input: Node) -> ParseResult<Command> {
+        if input.as_str() == "where \\u" {
+            Ok(Command::Where(true))
+        } else {
+            Ok(Command::PrintPC)
```

Comment:
Unfortunately this won't actually work since the pest grammar means that
comm_where is currently only valid if it is `where` or `pc` and it won't parse
the additonal command correctly. You'll need to modify the pest syntax file for
this one! Shoot me a message and I can walk you through how it works